app/(auth)/actions.ts

```diff
@@ -51,15 +51,15 @@ export async function signup(formData: FormData) {
   redirect("/");
 }
 
-// Google OAuth sign-in
-export async function signinWithGoogle() {
-
+// OAuth sign-in with Google or GitHub
+export async function signinWithOAuth(provider: 'google' | 'github') {
```

Comment:
Let's make an enum for providers

E.g.
```
enum Colors {
  RED = 'RED',
  GREEN = 'GREEN',
  BLUE = 'BLUE'
}

// Usage
const myColor: Colors = Colors.RED;
```

Reply:
Now using built-in supabase `Provider` type: https://github.com/trypear/pear-landing-page/pull/38/commits/9473ed1d363e085e07f262fff4b482d2e1ab4e66